source/help/apps/desktop-changelog.rst

```diff
@@ -1,6 +1,97 @@
 Desktop Application Changelog
 ========================================
 
+Release v4.3.0
+----------------------------
+
+- **Release date:** October 16, 2019
+- **Download Binary:** 
+
+**Note:** Mattermost v4.3.0 contains a medium level security fix. `Upgrading <http://docs.mattermost.com/administration/upgrade.html>`__ is highly recommended. Details will be posted on our `security updates page <https://about.mattermost.com/security-updates/>`__ 30 days after release as per the `Mattermost Responsible Disclosure Policy <https://www.mattermost.org/responsible-disclosure-policy/>`__.
+
+**Breaking Change** 
+
+The Mattermost Desktop v4.3.0 release includes a change to how desktop notifications are sent from non-secure URLs (http://). Organizations using non-secure Mattermost Servers (http://) will need to update to Mattermost Server versions 5.16.0+, 5.15.1, 5.14.4 or 5.9.5 (ESR) to continue receiving desktop notifications when using Mattermost Desktop v4.3.0 or later.
+
+Improvements
+~~~~~~~~~~~~~~~
+
+All Platforms
+^^^^^^^^^^^^^
+
+- Added support for maintaining a user's online status while the desktop app is in the background as well as automatically setting the user to away/online if the screensaver goes on/off, the user locks/unlocks their computer or shuts down their computer.
```

Comment:
Added support for maintaining a user's online status while the desktop app is in the background but the user is interacting with their computer